### ISSUE TITLE
Request for config for training for Real3D-AD dataset

### ISSUE THREAD

**Issue by alibalapour** [hzzzzzhappy/MC4AD#2]
Hi,
Thank you for sharing your work's code.
Can you please share the config that you have used for training on the Real3D-AD dataset?

**Comment by hzzzzzhappy** [comment]
Hi,


Thank you for your attention. Requires more warm-up on real3D and a slower learning rate scheduler. We will further maintain the code base after the paper is accepted. Until then, it's hard to find time to update this library. Because we made minor adjustments to each parameter on real3dad, we adjusted it manually during the initial training and did not record it.


Before we do that, we introduce you to another warehouse that shows a complete configuration that you can view UCA-3DL：https://github.com/M-3LAB/UCA-3DL

This code is based on the same PO3AD improvements as this repository, but is more extensible and performs well on multi-class anomaly detection.


Thank much for your understanding.




--------------原始邮件--------------
发件人："Ali&nbsp;Balapour ***@***.***&gt;;
发送时间：2026年1月5日(星期一) 凌晨4:16
收件人："hzzzzzhappy/MC4AD" ***@***.***&gt;;
抄送："Subscribed ***@***.***&gt;;
主题：[hzzzzzhappy/MC4AD] Request for config for training for Real3D-AD dataset (Issue #2)
-----------------------------------

 alibalapour阿利巴拉普尔 created an issue 创建了一个问题(hzzzzzhappy/MC4AD#2)(hzzzzzhappy/MC4AD#2)

Hi,你好，
 Thank you for sharing your work's code.谢谢你分享你的作品代码。
 Can you please share the config that you have used for training on the Real3D-AD dataset?您能否分享您在&nbsp;Real3D-AD数据集上进行培训时使用的配置？

—&amp;mdash;

**Comment by alibalapour** [comment]
Thank you very much for your answer. Best of luck with your submission.

**Comment by alibalapour** [comment]
Sorry to reopen this issue, I have another question regarding how you trained your model on the Real3D-AD dataset. In this dataset, the training set consists of full 360° scans of objects, while the test set contains only single-view scans, which introduces a domain shift between the train and test sets.

How do you solve this issue? Is your model robust to this discrepancy?

**Comment by hzzzzzhappy** [comment]
Hi,

Thank you for your renewed interest. For this matter, you may refer to the response provided in our other paper [ISMP](https://github.com/M-3LAB/Look-Inside-for-More/issues/8#issuecomment-3685543737).

In fact, our paper can follow the same configuration. If you adopt the approach for handling the training set outlined in this answer: [GLFM](https://github.com/hustCYQ/GLFM-Multi-class-3DAD), your performance may well improve.

Our experiments indicate that processing the training set does indeed yield a significant performance improvement [UCA-3DAL](https://www.researchgate.net/publication/398446535_A_Unified_Contrastive_Framework_with_Test-Time_Adaptation_for_Robust_3D_Anomaly_Localization)

**Comment by alibalapour** [comment]
Thanks!
